volatility3/framework/plugins/windows/dumpfiles.py

```diff
@@ -53,6 +54,15 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 description="Dump a single _FILE_OBJECT at this physical address",
                 optional=True,
             ),
+            requirements.StringRequirement(
```

Comment:
So adding filtering options like this is great, but also needs some consideration.  Is it consistent with other filter options?  `regex` isn't that descriptive a name, I'd prefer `filter` or `match` or something like that, and then let the `--help` describe that it's a regular expression.  Most people will know what regexs are, but not everybody, and it doesn't really describe the purpose of it.

Reply:
Consideration also needs to be used to determine how this option interacts with others?  If people provide `virtaddr` and `regex`, which wins? (or are they both applied?  Does that make sense?)  What about `pid`?  It should be clear and/or obvious which options will interact with each other and which cancel others out.

Reply:
The flag name change definitely makes sense and `--filter` makes a lot of sense, so I think changing to that will be good.

In terms of how it will interact with the other options, I think it makes sense that it cannot interact with the flags that take in an address, but it can work with the `pid` flag. My newest commit addresses this.